zproject/urls.py

```diff
@@ -716,6 +722,10 @@
     # Used to join a Big Blue Button video call
     path('calls/bigbluebutton/join', join_bigbluebutton),
 
+    # Used to generate a Webex video call URL
+    path('calls/webex/register', register_webex_user),
+    path('calls/webex/complete', complete_webex_user),
```

Comment:
Maybe put this above the BigBlueButton endpoints since it has a more similar model to Zoom?